ext/crates/fp/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![feature(const_panic)]
 #![allow(clippy::many_single_char_names)]
 #![allow(clippy::unreadable_literal)]
+#![feature(stdsimd)]
```

Comment:
I believe `stdsimd` is already stabilized, so this is not needed.
```suggestion
```

Reply:
I thought so too but AVX512 is still experimental. I realized that when I tested the code on the Wayne grid.

Reply:
That makes sense. I believe you can compile to AVX-512 even if your computer doesn't support it.